<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAPlot</class>
 <widget class="QDialog" name="CAPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CA Plot</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QPushButton" name="calcBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Plot Settings</string>
   </property>
   <widget class="QComboBox" name="selUnits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleLedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Units</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="plotArea"/>
  </widget>
  <widget class="QWidget" name="plotToolbar" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Concentration-Area Plot Creator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Thresholds</string>
   </property>
   <widget class="QPushButton" name="thrBtn3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thrBtn2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thrBtn1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshpoint1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshpoint2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshpoint3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick threshold values from chart</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="thrChk2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="thrChk3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="threshplotBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Threshold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thrChk1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Raster</string>
   </property>
   <widget class="QLineEdit" name="layerLedit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fileRbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="layerRbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From Layer</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="layerCbx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="layerRef">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="plotrefBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Plot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>thrChk2</sender>
   <signal>toggled(bool)</signal>
   <receiver>thrBtn2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk2</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshpoint2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk3</sender>
   <signal>toggled(bool)</signal>
   <receiver>thrBtn3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk3</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshpoint3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerLedit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerRbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerCbx</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerRbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerRef</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk1</sender>
   <signal>toggled(bool)</signal>
   <receiver>thrBtn1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk1</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshpoint1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk1</sender>
   <signal>toggled(bool)</signal>
   <receiver>thrChk2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thrChk2</sender>
   <signal>toggled(bool)</signal>
   <receiver>thrChk3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CAPlot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
